content/docs/examples/multicluster/icp/index.md

```diff
@@ -0,0 +1,219 @@
+    
+## Config Pod Communication Cross IBM Cloud Private Clusters
+
+IBM Cloud Private is using Calico Node-to-Node Mesh by default to manage container networks. The BGP Client
+on each node distributes the IP router information to all nodes.
+
+In order to make sure pods can communicate across different clusters, you need to configure IP routers on
+all nodes in the cluster. You need to add IP routers in cluster-2 to cluster-1, and add IP routers in
+cluster-1 to cluster-2.
+
+You can check how to add IP routers from cluster-1 to cluster-2. With Node-to-Node Mesh mode, each node will
+have IP routers connecting to peer nodes in the cluster. In this example, both cluster has three nodes.
+
+For cluster-1, the `hosts` file is as follows:
+
+{{< text bash >}}
+9.111.255.21 gyliu-icp-1
+9.111.255.129 gyliu-icp-2
+9.111.255.29 gyliu-icp-3
+{{< /text >}}
+
+For cluster-2, the `hosts` file is as follows:
+
+{{< text bash >}}
+9.111.255.152 gyliu-ubuntu-3
+9.111.255.155 gyliu-ubuntu-2
+9.111.255.77 gyliu-ubuntu-1
+{{< /text >}}
+
+1.  Get all of the `ip route` on nodes in cluster-1 with command `ip route | grep bird`. See the following
+    example. In this example, both cluster has three nodes.
+
+    {{< text bash >}}
+    root@gyliu-icp-1:~# ip route | grep bird
+    10.1.43.0/26 via 9.111.255.29 dev tunl0 proto bird onlink
+    10.1.158.192/26 via 9.111.255.129 dev tunl0 proto bird onlink
+    blackhole 10.1.198.128/26 proto bird
+    {{< /text >}}
+    {{< text bash >}}
+    root@gyliu-icp-2:~# ip route | grep bird
+    10.1.43.0/26 via 9.111.255.29 dev tunl0  proto bird onlink
+    blackhole 10.1.158.192/26  proto bird
+    {{< /text >}}
+    {{< text bash >}}
+    10.1.198.128/26 via 9.111.255.21 dev tunl0  proto bird onlink
+    root@gyliu-icp-3:~# ip route | grep bird
+    blackhole 10.1.43.0/26  proto bird
+    10.1.158.192/26 via 9.111.255.129 dev tunl0  proto bird onlink
+    10.1.198.128/26 via 9.111.255.21 dev tunl0  proto bird onlink
+    {{< /text >}}
+
+1.  You can see there are three IP routers total for those three nodes in cluster-1.
+
+    {{< text bash >}}
+    10.1.158.192/26 via 9.111.255.129 dev tunl0  proto bird onlink
+    10.1.198.128/26 via 9.111.255.21 dev tunl0  proto bird onlink
+    10.1.43.0/26 via 9.111.255.29 dev tunl0  proto bird onlink
+    {{< /text >}}
+
+1.  Add those three IP routers to all nodes in cluster-2 by the command as follows.
+
+    {{< text bash >}}
+    $ ip route add 10.1.158.192/26 via 9.111.255.129 
+    $ ip route add 10.1.198.128/26 via 9.111.255.21
+    $ ip route add 10.1.43.0/26 via 9.111.255.29
+    {{< /text >}}
+
+1.  You can use same steps to add all IP routers from cluster-2 tocluster-1, as well. After the configuration
+    finished, you will see that the pods in those two different clusters can communication with each other.
+
+1.  You can verify it by pinging pod IP in cluster-2 from cluster-1. The following is a pod from cluster-2 with
```

Comment:
You can verify it -> Verify cross pod communication

Reply:
Done